src/modules/Dashboard/pages/Applet/AppletMultiInformant.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { styled, Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+import { StyledFlexTopCenter, theme, variables } from 'shared/styles';
+
+export const StyledActions = styled(StyledFlexTopCenter)`
+  gap: ${theme.spacing(1)};
+`;
+
+export const StyledExportButton = styled(Button)`
```

Comment:
This feels like such minimal styles that it feels excessive to add a new Styled component, let alone a new file. Is there anything I can use instead to get rid of this file?

Reply:
@sabrichu I think the use of the [`sx`](https://mui.com/system/getting-started/the-sx-prop/) property could work, it's being used in other components as convention and it was suggested by @IhorNesterenko99 in previous PR reviews.

```tsx
<Button sx={{ gap: theme.spacing(1), color: variables.palette.on_surface_variant }} />
```

Reply:
Usually, we use sx instead of styled-components if the number of styles is no more than two.

Reply:
Sounds good! I'll do the same with `StyledSettingsButton`

Reply:
@IhorNesterenko99 Love it! I make the cut!!